subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -1378,6 +1392,10 @@ void ticker_worker(void *param)
 #else  /* CONFIG_BT_TICKER_LOW_LAT ||
 	* CONFIG_BT_TICKER_SLOT_AGNOSTIC
 	*/
+		/* Pick the ticker expiry latency, to calculate ticks_at_expire
+		 */
```

Comment:
```suggestion
		/* Pick the ticker expiry latency, to calculate ticks_at_expire */
```

Reply:
Looks like you've still (mis?)configured your editor to use 80 instead of 100 line length with autoformatting.